src/core/Akka.Cluster.Tests.MultiNode/SurviveNetworkInstabilitySpec.cs

```diff
@@ -135,7 +136,7 @@ private void AssertCanTalk(params RoleName[] alive)
             {
                 foreach (var to in alive)
                 {
-                    var sel = Sys.ActorSelection(Node(to) / "user" / "echo");
+                    var sel = Sys.ActorSelection(new RootActorPath(GetAddress(to)) / "user" / "echo");
```

Comment:
`GetAddress` uses caching under the hood, so we can save ourselves some network calls here